content/post/what-is-nodejs-loaders.mdx

```diff
@@ -1,10 +1,26 @@
 ---
 title: What is Nodejs-Loaders
-description: Nodejs-Loaders is a collection of loaders for Node.js.
+description: nodejs-loaders is a project designed to provide a variety of loaders that make local development and CI testing quicker and easier.
 authors: AugustinMauroy, JakobJingleheimer
 date: 2025-07-01
 category: information
 ---
 
+## Our Values: Simplicity, Lightness, Efficiency
+
+We believe in keeping things simple and efficient. nodejs-loaders is built with these core values in mind:
+
+* **Simplicity:** Easy to use and understand.
+* **Lightness:** Minimal overhead and dependencies.
+* **Efficiency:** Optimized for speed and performance.
```

Comment:
```suggestion
We believe in keeping things simple and efficient. nodejs-loaders is built with these core values in mind: Simplicity and Efficiency.
```

Lightness isn't really a word 😅

I think we don't need to define what simplicity and efficiency mean.